royalties-registry/test/contracts/RoyaltiesRegistry.test.js

```diff
@@ -1,265 +0,0 @@
-const { deployProxy, upgradeProxy } = require('@openzeppelin/truffle-upgrades');
-const RoyaltiesRegistry = artifacts.require("RoyaltiesRegistry.sol");
-const RoyaltiesRegistryTest = artifacts.require("RoyaltiesRegistryTest.sol");
-const TestERC721 = artifacts.require("TestERC721.sol");
-const TestERC721RoyaltyV1OwnUpgrd = artifacts.require("TestERC721WithRoyaltiesV1OwnableUpgradeable");
-const TestERC721RoyaltyV2OwnUpgrd = artifacts.require("TestERC721WithRoyaltiesV2OwnableUpgradeable");
-const TestRoyaltiesProvider = artifacts.require("RoyaltiesProviderTest.sol");
-const truffleAssert = require('truffle-assertions');
-
-const { expectThrow, verifyBalanceChange } = require("@daonomic/tests-common");
-
-contract("RoyaltiesRegistry, test methods", accounts => {
```

Comment:
could you move this file and not delete and create? This way history will be preserved